IdokladSdk/Requests/Core/Modifiers/Expand/Structure/ReceivedReceiptExpand.cs

```diff
@@ -0,0 +1,44 @@
+﻿using IdokladSdk.Models.BankStatement;
+using IdokladSdk.Models.CashVoucher;
+using IdokladSdk.Models.ReadOnly.Currency;
+using IdokladSdk.Models.ReadOnly.PaymentOption;
+using IdokladSdk.Requests.Core.Modifiers.Expand.Common;
+
+namespace IdokladSdk.Requests.Core.Modifiers.Expand.Structure
+{
+    /// <summary>
+    /// ReceivedReceiptExpand.
+    /// </summary>
+    public class ReceivedReceiptExpand : ExpandableEntity
+    {
+        /// <summary>
+        /// Gets or sets the associated bank statement.
+        /// </summary>
+        public BankStatementGetModel BankStatement { get; set; }
+
+        /// <summary>
+        /// Gets or sets the associated cash voucher.
+        /// </summary>
+        public CashVoucherGetModel CashVoucher { get; set; }
+
+        /// <summary>
+        /// Gets or sets the associated currency.
+        /// </summary>
+        public CurrencyGetModel Currency { get; set; }
+
+        /// <summary>
```

Comment:
Tyto dve maji byt v expandu?

Items, Partner

Items jsem jinde nenasel ze by byly

Reply:
JJ melo by to tam byt. Items mame i na received invoice

Reply:
No je je divne protoze treba IssuedInvoiceExpand, CreditNoteExpand to nemají tak nechybí to tam nahodou?

Reply:
No podle me to tam chybi, protoze si pak nemuze expandnout items

Reply:
Pridano